### ISSUE TITLE
Migrate Janus static config files to uStreamer Debian package

### ISSUE THREAD

**Issue by jdeanwallace** [tiny-pilot/ustreamer-debian#12]
Related https://github.com/tiny-pilot/tinypilot/issues/1353
Blocked on https://github.com/tiny-pilot/tinypilot/issues/1427

Migrate [this uStreamer Ansible task](https://github.com/tiny-pilot/ansible-role-ustreamer/blob/02f37b869c8c585517a9c7b6cd95cfb7661f1490/tasks/install_janus.yml#L33-L44) to our uStreamer Debian package.

**Comment by mtlynch** [comment]
Closing this in favor of https://github.com/tiny-pilot/tinypilot/issues/1496 as I'd like it to be in the TinyPilot Debian package
